src/main/fc/fc_tasks.c

```diff
@@ -452,6 +460,13 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .staticPriority = TASK_PRIORITY_MEDIUM,
     },
 
+    [TASK_TEMPERATURE] = {
+        .taskName = "TEMPERATURE",
+        .taskFunc = taskUpdateTemperature,
+        .desiredPeriod = TASK_PERIOD_HZ(10),      // 10 Hz
```

Comment:
I'm pretty sure it can go down to 1Hz and no one would notice

Reply:
Yeah, probably. Temperature inertia is quite large.